### DISCUSSION TITLE
Search bar font color very difficult to read

### DISCUSSION THREAD

**Discussion by Sagnasty** [rern/rAudio#410] [Fixed Issues]
Font color is almost the same as the background (Running version rAudio 1 202106191).

Jim

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

![font color](https://user-images.githubusercontent.com/76575081/122684775-8d12f080-d1d5-11eb-9dc9-f7439d2f2fa6.jpg)

**Comment by AndyFx12** [comment]
Same here. Chrome on Android.

Andy

**Comment by Sagnasty** [comment]
Confirmed fixed in rAudio 1 20210621

Jim

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+
